### ISSUE TITLE
Review of the lab9 solution - done by Francesca Zafonte

### ISSUE THREAD

**Issue by Zafonte** [luanapulignano/Computational-Intelligence#2]
Hi Luana, I reviewed your code and I'm glad to share my feedback. 

_**README:**_
I simply suggest adding more explanation of the problem you are solving and why certain choices were   made.

_**CODE:**_
- The code is well formatted and easily readable. The names of variables, functions are descriptive.
I would just add a few more comments.

- Recombination: it would be interesting to implement other types of crossovers such as uniform_xover()

- To reduce the number of fitness calls I think that you could keep track of the assessments already carried out in order to avoid duplicate assessments.

- I would suggest to add some graphs in order to have a vision of the results obtained.

Unfortunately I have not implemented my own solution to the problem so I have no suggestions to give you for better results.
To summarize, I think you did a good implementation and I hope that you'll find my little tips useful!
